src/datalad_installer.py

```diff
@@ -1813,6 +1821,17 @@ def ask(prompt: str, choices: List[str]) -> str:
             return answer
 
 
+def get_version_codename() -> str:
+    with open("/etc/os-release") as fp:
+        for line in fp:
+            m = re.fullmatch(
+                r'VERSION_CODENAME=(")(?P<value>[^"]+)(?(1)"|)', line.strip()
```

Comment:
ha -- good find, forgot about that one, but 

<details>
<summary>none on debian sid (unstable) or testing (current bullseye) </summary> 

```shell
$> docker run --rm debian:sid cat /etc/os-release      
Unable to find image 'debian:sid' locally
sid: Pulling from library/debian
e9e6a013db8a: Pull complete 
Digest: sha256:4ae5284ee9adbb10397cbc5343aaaafa34ff9f58f6d5c51589d3ba41ddcae4df
Status: Downloaded newer image for debian:sid
PRETTY_NAME="Debian GNU/Linux bullseye/sid"
NAME="Debian GNU/Linux"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

```
</details>
so wouldn't be sufficient alone

Reply:
It appears that, on Debian systems that don't have that value in `/etc/os-release`, `/etc/debian_version` can be consulted instead.

Reply:
ok, FWIW found no corner cases for that